code/index.html

```diff
@@ -3,39 +3,66 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="This is a boilerplate for building websites"/>
+    <meta name="beached whale" content="beached whale adventure apparel and knickknacks"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Trad Is Rad</title>
-    <link href='http://fonts.googleapis.com/css?family=Amatic+SC' rel='stylesheet' type='text/css'>
+
+  <!--- FONTS -------------------->
+    <link href="http://fonts.googleapis.com/css?family=Amatic+SC:400,700" rel="stylesheet" type="text/css">
```

Comment:
When possible use single quotes instead of double quotes. This makes the file size smaller.